### ISSUE TITLE
Ace is not defined error

### ISSUE THREAD

**Issue by richardartoul** [Vertamedia/clickhouse-grafana#148]
When I try to create a new data source of type Clickhouse I get the following error

<img width="996" alt="Screen Shot 2019-10-11 at 1 51 39 PM" src="https://user-images.githubusercontent.com/9171254/66673306-47470c00-ec2e-11e9-8669-7d2fd9fb5e98.png">

Can be reproduced with the following Dockerfile:

```
FROM grafana/grafana:latest

RUN grafana-cli plugins install vertamedia-clickhouse-datasource
```

**Comment by amonakhov** [comment]
There is a release in place already: https://github.com/Vertamedia/clickhouse-grafana/releases/tag/1.9.2

And there is a pull request for Grafana as well: https://github.com/grafana/grafana-plugin-repository/pull/496

**Comment by Slach** [comment]
I close this issue cause it inactive and already solved in version 1.9.2, feel free reopen if necessary
